.github/workflows/memleaks.yml

```diff
@@ -0,0 +1,49 @@
+name: memleaks
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+env:
+  PREFIX: $HOME/build/
```

Comment:
This is no longer valid due to changes to the cmake call / build.sh recipe

Reply:
discussed in newer comment